editions/tw5.com/tiddlers/howtos/Custom tag-pill styles by data-tag-title.tid

```diff
@@ -0,0 +1,32 @@
+created: 20230608121519758
+modified: 20230608123444591
+tags: [[How to apply custom styles]]
+title: Custom tap-pill styles by data-tag-title
```

Comment:
Should be "tag", not "tap". Also don't need the dash. And I'm not sure that the title needs to mention "data-tag-title".

So, perhaps just `Custom tag pill styles`